sim/warrior/whirlwind.go

```diff
@@ -18,8 +18,8 @@ func (warrior *Warrior) RegisterWhirlwindSpell() {
 		whirlwindOH = warrior.RegisterSpell(core.SpellConfig{
 			ActionID:       actionID.WithTag(1),
 			SpellSchool:    core.SpellSchoolPhysical,
-			ProcMask:       core.ProcMaskEmpty, // whirlwind offhand hits usually don't proc auras
-			ClassSpellMask: SpellDualWieldSpecMask,
+			ProcMask:       core.ProcMaskMeleeOHSpecial, //TODO: needs testing to check if it procs auras, according to pre-cata it didn't
```

Comment:
If this comment turns out correct then you can add a 2nd class spell mask for WW oh spell and apply a 2nd mod with the same bonus just for it. If after testing WW OH hits should proc stuff then you are already covered.